assets/logic.js

```diff
@@ -1,4 +1,29 @@
 $('.jsToggle').on('click', function(){
```

Comment:
While your in this file can you stick

`'use strict';` at the top

This will flag up a couple of the errors in this file automatically.